### PR TITLE
Switch to moveit_ci, apply clang-format

### PR DESCRIPTION
**Pull request by davetcoleman**
This repo was using a bad CI approach and not using our best practices

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,32 +1,26 @@
...
-  - generate_devel_script.py https://raw.githubusercontent.com/ros-infrastructure/ros_buildfarm_config/production/index.yaml $ROS_DISTRO_NAME default $REPOSITORY_NAME $OS_NAME $OS_CODE_NAME $ARCH > $JOB_PATH/devel_job.sh
-  - cd $JOB_PATH
-  - cat devel_job.sh
-  # run the actual job which involves Docker
-  - sh devel_job.sh -y
+    - TEST="clang-format catkin_lint"
+    - TEST=clang-tidy-fix
+    - DOCKER_IMAGE=moveit/moveit:melodic-source
+    - DOCKER_IMAGE=moveit/moveit:master-source
+    - DOCKER_IMAGE=moveit/moveit:kinetic-ci  UPSTREAM_WORKSPACE=moveit.rosinstall  TEST_BLACKLIST=moveit_ros_perception
```
Comment (henningkayser) [inline .travis.yml]: I think this should  use `rviz_visual_tools.rosinstall` which doesn't exist yet

Comment (davetcoleman) [reply .travis.yml]:
```suggestion
    - DOCKER_IMAGE=moveit/moveit:kinetic-ci  TEST_BLACKLIST=moveit_ros_perception
```

**Comment by davetcoleman** [comment]
There are a lot of clang-tidy errors, which I do not have time for :-/
I think this PR will just stay open until someone is able to address this...

**Comment by davetcoleman** [comment]
I've actually just removed the clang-tidy tests so we can merge this in now...

**Comment by davetcoleman** [comment]
I've added to JIRA

**Pull request merged**